### ISSUE TITLE
Document how to start the lsp java server

### ISSUE THREAD

**Issue by naclander** [emacs-lsp/lsp-java#3]
Similar to #1 I think we should  document how to start the language server from within emacs. 

I have the server downloaded and in the correct location, and I can start it manually, but I am unsure how to do this within emacs using lsp-mode and lsp-java.

**Comment by naclander** [comment]
It seems that the java lsp server does start *sometimes*. Its hard for me to diagnose why it starts some times and not others. 

I like the path that [meghanada](https://github.com/mopemope/meghanada-emacs)  took and had all server interaction logged to a separate buffer.

**Comment by vibhavp** [comment]
lsp-mode should start the language server automatically. Is that not happening?

**Comment by vibhavp** [comment]
Also, I like the idea to log all server interaction. Structured logging for requests and response might be a feature in lsp-mode at some point.

**Comment by naclander** [comment]
Actually I seem to sometimes get a prompt to start the lsp-server, but depending on how I open the buffer it sometimes doesn't get prompted ( for example, if I open a file via neotree ). 

This might be a problem with lsp-mode and not lsp-java, but it would still be nice if I had a command to force the java lsp server to start ( and maybe restart and kill it as well ).

**Comment by yyoncho** [comment]
In order to force `lsp-java` server to start, you could call `lsp-java-enable` manually. You could restart the workspace with `lsp-restart-workspace`.

**Comment by ivanhjc** [comment]
> In order to force `lsp-java` server to start, you could call `lsp-java-enable` manually. You could restart the workspace with `lsp-restart-workspace`.

Is `lsp-java-enable` unavailable now? I'm using lsp-java 20191115.1712 and it's not found when I try to call it with `M-x`. I tried `lsp-restart-workspace` and it let me select a workspace to restart while there's no matching workspace. I also tried to add a workspace by calling `lsp-workspace-folders-add` but still no good. As of now what's the proper way to start the server from within Emacs?
